### PR TITLE
at: add more information link

### PR DESCRIPTION
**Pull request by patricedenis**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #5510

### CODE REVIEW

**Comment by marchersimon** [comment]
Thanks!
 Could you please also add the link to the Brazilian Portugese (pt_BR) and the Chinese (zh) pages?
`More information:` in Brazilian Portugese is `Mais informações:` and in Chinese is `更多信息：`

**Comment by patricedenis** [comment]
> Thanks!
> Could you please also add the link to the Brazilian Portugese (pt_BR) and the Chinese (zh) pages?
> `More information:` in Brazilian Portugese is `Mais informações:` and in Chinese is `更多信息：`

done !

**Review by marchersimon — APPROVED** [review]

File: pages.pt_BR/common/at.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 > Ferramenta para o agendamento de comandos.
```
Comment (navarroaxel) [inline pages.pt_BR/common/at.md:3]: Why this page is in the `common/` folder?

File: pages.fr/linux/at.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# at
+
+> Planifie l'exécution d'une commande une fois à un moment donné.
```
Comment (navarroaxel) [inline pages.fr/linux/at.md:3]: What about this page https://github.com/tldr-pages/tldr/blob/master/pages.fr/common/at.md ?

File: pages.pt_BR/common/at.md (hunk 1 above)
Comment (marchersimon) [reply pages.pt_BR/common/at.md:3]: There is a variant in `common/` and `linux/`.

File: pages.fr/linux/at.md (hunk 2 above)
Comment (marchersimon) [reply pages.fr/linux/at.md:3]: There's also a `it`, `ko`, `pl` and `pt_BR` version of `at.md` in `common/`. Should we maybe do that in a seperate PR?

Comment (navarroaxel) [reply pages.fr/linux/at.md:3]: Maybe the `linux/` and `common/` folders should be unified, I don´t know if the parameters are different but sounds strange to me. @bl-ue could you check the command in macOS?

File: pages.pt_BR/common/at.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
...
 > O serviço atd (ou atrun) deve estar sendo executado para as atuais execuções.
+> Mais informações: <https://www.man7.org/linux/man-pages/man1/at.1p.html>.
 
 - Executar comandos da standard input em 5 minutos (pressionar `Ctrl + D`quando acabar):
```
Comment (bl-ue) [inline pages.pt_BR/common/at.md:7]:
```suggestion
- Executar comandos da standard input em 5 minutos (pressionar `Ctrl + D` quando acabar):
```

File: pages.fr/linux/at.md (hunk 2 above)
Comment (bl-ue) [reply pages.fr/linux/at.md:3]: Yeah I have `at`, here's the man page:
```
at, batch, atq, atrm -- queue, examine, or delete jobs for later execution

at [-q queue] [-f file] [-mldbv] time
at [-q queue] [-f file] [-mldbv] -t [[CC]YY]MMDDhhmm[.SS]
at -c job [job ...]
at -l [job ...]
at -l -q queue
at -r job [job ...]

atq [-q queue] [-v]

atrm job [job ...]

batch [-q queue] [-f file] [-mv] [time]
```

**Comment by bl-ue** [comment]
@patricedenis thank you for this PR, but I'm going to close it for two reasons:

1. I'm opening a PR that adds a more info link to `at`, `atq`, `atrm`, and `batch`
2. As @navarroaxel [pointed out](https://github.com/tldr-pages/tldr/pull/5540#discussion_r603967605), we already have a French translation of `common/at.md` located [here](https://github.com/tldr-pages/tldr/blob/161a1d836432d9243dcf4fcf03e1289344ac23df/pages.fr/common/at.md).

Thank you anyway for it, though! ❤️

**Comment by patricedenis** [comment]
yeah, I didn't undestand why is there several commands that can be found in both linux and common platform. Is that normal?

**Comment by bl-ue** [comment]
So it looks like it was an oversight. #929 added `linux/at.md`, and #1344 added `common/{at,atq,atrm,batch}.md`.  I'll merge them both into `common/at.md` 👍🏻

**Comment by patricedenis** [comment]
@bl-ue great !
Thanks !

**Pull request closed without merging**